test/e2e/assets/test.js

```diff
@@ -89,11 +89,25 @@ test('`fusion dev` works with assets', async () => {
     );
     t.equal(await request(`${url}/dirname`), 'src');
     t.equal(await request(`${url}/filename`), 'src/main.js');
+
+    const jsonAssetUrl = await request(`${url}/json`);
+    const jsonAsset = (await request(url + jsonAssetUrl)).replace(/\s/g, '');
```

Comment:
Nitpick: maybe use `JSON.parse` and `t.deepEqual` instead?

Reply:
Much better. Thanks!